sdk/src/assertions/metadata.rs

```diff
@@ -248,7 +248,7 @@ pub enum ReviewCode {
     Other(String),
 }
 
-/// A rating on an [`Assertion`].
+/// A rating on an Assertion.
```

Comment:
Because this references the low level internal Assertion which the new API no longer exposes. We expose ManifestAssertion, so I could reference that here. Ideally we would rename ManifestAssertion to Assertion in the public interface, but that would have backward compatibility issues.